packages/api/src/routes-v1/nfts-store.js

```diff
@@ -0,0 +1,93 @@
+import { validate } from '../utils/auth-v1.js'
+import setIn from 'just-safe-set'
```

Comment:
Do we need to port the fix from https://github.com/ipfs-shipyard/nft.storage/pull/343?